pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "flit_core.buildapi"
 [project]
 # Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
 name = "ansys-tools-path"
-version = "0.4.dev0"
+version = "0.4.dev1"
```

Comment:
Why changing the dev version here? Should be 0 unless we do a dev release

Reply:
@RobPasMue In this process, is it possible to publish a beta (dev) version of the package? I would need to use this package from a GitHub workflow and from an Azure pipeline.  Are there some functionalities in the package that are not tested and that block the release of the package?

Reply:
@nmalsang I think you need to make the functional change in this PR without bumping the version, and then you can decide whether you want to do a minor release or a patch release separately

Reply:
I did not see that no package 0.4.dev0 was not publish. I thought it was publish each time. I revert this change.  @RobPasMue if it is OK for you I gonna publish the package when the PR will be reviewed.

Reply:
Process is as follows:
1) Submit a PR with your functional changes
2) Decide whether you want to do a patch or a minor release
2.1) In case of a patch release, cherry-pick this commit into the release/0.4 branch, bump the version, create a new tag and push the changes and the tag
2.2) In case of a minor release, create a new branch release/0.5 from main, change the version, create a new tag and push the changes and the tag. Afterwards, bump the version in the main branch